python/cuml/dask/common/input_utils.py

```diff
@@ -255,3 +257,8 @@ def to_dask_cupy(futures, dtype=None, shapes=None, client=None):
             objs.append(obj)
 
     return da.concatenate(objs, axis=0)
+
+
+def wait_and_raise_from_futures(futures):
```

Comment:
Should have a docstring.
Also, probably return the result of wait just in case the user needs it.